cloudwatch/cloudwatch.go

```diff
@@ -102,6 +103,7 @@ type OutputPluginConfig struct {
 	RoleARN          string
 	AutoCreateGroup  bool
 	CWEndpoint       string
+	STSEndpoint      string
```

Comment:
I noticed we don't have an `OutputPluginConfig` struct in our other plugins. I think we should be consistent across the plugins, so maybe we should add items to the other plugin repos to adopt this pattern?

Reply:
We should, the config struct is a lot cleaner. I don't want to do that refactoring right now though...